src/components/Tabs.tsx

```diff
@@ -25,13 +29,32 @@ function Tabs({ definition }: TabsProps) {
           path={`${path}${url}`}
           key={url}
           children={
-            <div className={styles.Tabs}>
-              <TabBar tabs={tabs} current={url} path={path} />
-              <div className={styles.TabContent}>{component}</div>
-            </div>
+            type === "floodsmart" ? (
+              <div className={styles.Tabs}>
+                <TabBar tabs={tabs} current={url} path={path} />
+                <div className={styles.TabContent}>{component}</div>
+              </div>
+            ) : (
+              <Flex>
+                <ConfigTabBar tabs={tabs} current={url} path={path} />
+                <Box flex="1" marginLeft="8">
+                  {component}
+                </Box>
+              </Flex>
+            )
           }
         />
       ))}
+      {placeholder ? (
+        <Route path={path}>
+          <Flex>
+            <ConfigTabBar tabs={tabs} current="" path={path} />
+            <Box flex="1" marginLeft="4">
+              {placeholder}
+            </Box>
+          </Flex>
+        </Route>
+      ) : null}
```

Comment:
IMO, I think the ConfigTab can be made into a separate component as the Tabs component and the ConfigTab are not so similar to me. In this case, the ConfigTab can be edited and customised easier.

Reply:
Good point. I was in doubt a bit over this (because I also hadn't decided yet what the config tabs would look like), I'll refactor it.